src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -595,6 +596,20 @@ def load_command_table(self, _):
         g.custom_command('enable', 'server_aad_only_enable')
         g.show_command('get', 'get')
 
+    ###############################################
+    #           sql server trust groups           #
+    ###############################################
+
+    server_trust_groups_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerTrustGroupsOperations.{}',
+        client_factory=get_sql_server_trust_groups_operations)
+
+    with self.command_group('sql stg', server_trust_groups_operations, client_factory=get_sql_server_trust_groups_operations, is_preview=True) as g:
+        g.custom_command('create', 'server_trust_group_create')
+        g.custom_command('delete', 'server_trust_group_delete', confirmation=True)
+        g.custom_command('get', 'server_trust_group_get')
```

Comment:
recommend to use custom_show_command not custom_command.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands